letsencrypt-apache/letsencrypt_apache/constants.py

```diff
@@ -27,3 +27,15 @@
 REWRITE_HTTPS_ARGS = [
     "^", "https://%{SERVER_NAME}%{REQUEST_URI}", "[L,QSA,R=permanent]"]
 """Apache rewrite rule arguments used for redirections to https vhost"""
+
+
+HSTS_ARGS = ["always", "set", "Strict-Transport-Security",
+    "\"max-age=31536000; includeSubDomains\""]
```

Comment:
Is there any reason for this level of detail to be inside the plugins themselves?  In the long run my thought for the "right" way to do HSTS enablement in interactive mode would be to start it off with a very short TTL, and gradually increase it if the admin has left the header in place.  But freedom to implement that kind of functionality would require the "details" of the headers to be determined in the main client, not the plugins.

(I'm being a little picky about this because once we've picked a particular set of semantics for then "http-header" enhancement that essentially becomes an API between the client and plugins).